addons/l10n_se/data/account.account.template.csv

```diff
@@ -62,18 +62,17 @@
 "a2091","2091","Balanserad vinst eller förlust","account.data_account_type_equity","l10nse_chart_template","False"
 "a2098","2098","Vinst eller förlust från föregående år","account.data_account_type_equity","l10nse_chart_template","False"
 "a2099","2099","Årets resultat","account.data_account_type_equity","l10nse_chart_template","False"
-"a2123","2123","Periodiseringsfond 2013","account.data_account_type_equity","l10nse_chart_template","False"
 "a2124","2124","Periodiseringsfond 2014","account.data_account_type_equity","l10nse_chart_template","False"
 "a2125","2125","Periodiseringsfond 2015","account.data_account_type_equity","l10nse_chart_template","False"
 "a2126","2126","Periodiseringsfond 2016","account.data_account_type_equity","l10nse_chart_template","False"
 "a2127","2127","Periodiseringsfond 2017","account.data_account_type_equity","l10nse_chart_template","False"
 "a2128","2128","Periodiseringsfond 2018","account.data_account_type_equity","l10nse_chart_template","False"
 "a2129","2129","Periodiseringsfond 2019","account.data_account_type_equity","l10nse_chart_template","False"
-"a2130","2130","Periodiseringsfond 2020","account.data_account_type_equity","l10nse_chart_template","False"
-"a2131","2131","Periodiseringsfond 2021","account.data_account_type_equity","l10nse_chart_template","False"
-"a2132","2132","Periodiseringsfond 2022","account.data_account_type_equity","l10nse_chart_template","False"
-"a2133","2133","Periodiseringsfond 2023","account.data_account_type_equity","l10nse_chart_template","False"
-"a2134","2134","Periodiseringsfond 2024","account.data_account_type_equity","l10nse_chart_template","False"
+"a2130","2120","Periodiseringsfond 2020","account.data_account_type_equity","l10nse_chart_template","False"
```

Comment:
shouldn't the id match the code ? `a2130` and `2120`.

Reply:
updated the code